src/Search/Search.Tests/Tests/SearchIndexClientTests.cs

```diff
@@ -45,6 +45,58 @@ public void ConstructorCreatesCorrectUrl()
         }
 
         [Fact]
+        public void CanChangeIndexAfterConstructionWithoutServiceName()
+        {
+            Run(() =>
+            {
+                SearchServiceClient serviceClient = Data.GetSearchServiceClient();
+
+                // Create a second index.
+                Index index = serviceClient.Indexes.Get(Data.IndexName);
+                string newIndexName = index.Name + "2";
+                index.Name = newIndexName;
+
+                serviceClient.Indexes.Create(index);
+
+                // Make sure first index works.
+                SearchIndexClient indexClient = Data.GetSearchIndexClient();
+                indexClient.Documents.Count();
+
+                // Target the second index and make sure it works too.
+                indexClient.TargetDifferentIndex(newIndexName);
+                indexClient.Documents.Count();
```

Comment:
Good point. As you suggested, I'll delete the first index before testing the second one.